front/app/services/formCustomFields.ts

```diff
@@ -61,6 +61,38 @@ export type IFlatCustomFieldWithIndex = IFlatCustomField & {
   index: number;
 };
 
+const properties = [
+  'id',
+  'input_type',
+  'description_multiloc',
+  'required',
+  'title_multiloc',
+  'maximum_label_multiloc',
+  'minimum_label_multiloc',
+  'maximum',
+  'options',
+  'enabled',
+  'index',
+];
+
+const objectHasProperties = (
+  element: unknown,
+  propertyNames: string[]
+): boolean => {
+  let hasProperties = true;
+  propertyNames.forEach((propertyName) => {
+    if (!Object.prototype.hasOwnProperty.call(element, propertyName)) {
+      hasProperties = false;
+    }
+  });
+  return hasProperties;
+};
+
+export const isFlatCustomFieldWithIndex = (
```

Comment:
Minor: To make it easy for someone to know what we are doing, we could rename this to `isNewCustomFieldObject` or something similar. Only new fields have an index and that way the person reading the code will know even without reading the function

Reply:
I guess in this case though we're not specifically caring if it's a "new" one are we? We're just checking if an element is of the FlatCustomField type? Do you think we could use "isCustomFieldObject" instead?

Reply:
Ahh ok I see what you mean. I'll make the changes :) Thanks!